test/fixtures/r-httr2/get_charles_syntax.r

```diff
@@ -0,0 +1,10 @@
+library(httr2)
+
+request("http://localhost:28139/?format=json&") |> 
```

Comment:
Why doesn't this use `req_url_query()`? The trailing `&`?

Reply:
The trailing `&` seems to be an issue for the parser.
@verhovsky Why does `curl --compressed "http://localhost:28139/?format=json&"` result in a request with an empty query list but the query array is `format=json&`. Is that on purpose?

Reply:
We want to preserve the trailing `&` so that the generated code is actually equivalent to the input curl command.

queryList is empty because most HTTP libraries, if they have a way of passing the query as a list/dictionary, don't have a mechanism to send '' (does httr2?) without a =. If you do something like `params={'': ''}` in Python for example, that would send `format=json&=`. It's easier to catch this in the parser and not have a queryList at all, so that we fallback to requesting a raw URL, which would send the query as-is.

https://github.com/curlconverter/curlconverter/blob/master/src/Query.ts#L82

Reply:
Okay, makes sense, thanks!